README.md

```diff
@@ -1,58 +1,56 @@
-* Fast alle relevanten Regeln sind im Tool enthalten
+![Screenshot](docs/assets/images/readme_screenshot.png)
 
-Bei Fragen / Änderungswünschen / Feedback einfach Bescheid sagen - im dsaforum oder hier!
+## Features
+* ⚡ Dank Archetypen kann ein Charakter binnen Minuten erstellt werden
+* 📜 Relevante Regeln werden angezeigt und können der PDF angehängt werden
+* ⚖️ Die Regelbasis (Fertigkeiten, Vorteile etc.) kann frei editiert und erweitert werden
+* 🧩 Tierbegleiter, VTT-Exports, Manöverkarten und vieles mehr steht durch [Plugins](https://github.com/brzGatsu/SephrastoPlugins) bereit
+* ♻️ Steigerungen können rückgängig gemacht werden
+* 🤝 Von und für die Ilaris-Community entwickelt. Fragen und Vorschläge sind immer wilkommen: [DSA-Forum](https://dsaforum.de/viewtopic.php?t=45794&sid=f6d460e9749e5943fbd2c06b439bd46a) | [Discord](https://discord.gg/RsRcspq254)
+
+## Installation
+
+### Windows
+Für Windows stellen wir unter https://github.com/Aeolitus/Sephrasto/releases Builds zur Verfügung.
+
+### Linux
+Für Ubuntu und ähnliche Distributionen kannst du den automatischen Installer benutzen, indem du diesen Befehl in deinem Terminal ausführst:
+```bash
+curl -sSL https://raw.githubusercontent.com/Aeolitus/Sephrasto/master/tools/install_linux.sh | bash
+```
+Der Installer erstellt einen Sephrasto Ordner dort, wo er ausgeführt wird, lädt alle Dateien und Pakete die benötigt werden automatisch herunter und erstellt einen Eintrag in deinem Startmenü.
+Zum späteren Updaten kannst du den Ordner löschen und den Befehl erneut ausführen. Helden, Regeln und Plugins werden standardmäßig in einem anderen Ordner abgelegt.
 
-## Verwendete Pakete und Programme
-* Um Sephrasto mit Python zu verwenden, wird Version 3.11 empfohlen. Höhere Versionen sollten auch gehen, die Mindestversion ist 3.9.
-* PySide6 (https://www.qt.io/qt-for-python) für die Grafische Oberfläche
-* lxml (http://lxml.de/) als xml-parser (unter Linux: `libxml2-dev` und `libxmlsec1-dev` global installieren, wenn lxml via pip installiert werden soll)
-* PyYAML (https://pyyaml.org/) als yaml-parser
-* RestrictedPython (https://github.com/zopefoundation/RestrictedPython) zum Absichern der durch Benutzer erstellbaren Python Scripts
-* cx_freeze (optional, https://github.com/marcelotduarte/cx_Freeze) um Sephrasto-Builds zu erstellen
-* mkdocs (optional, https://github.com/mkdocs/) und mkdocs-material. Zum Konvertieren der Markdown-Dokumentation in HTML.
+Alternativ gibt es hier aus der Community einen Installer für Void Linux, Ubuntu/Debian und Arch Linux (ohne Gewähr): https://github.com/kgitthoene/multi-linux-sephrasto-installer
 
-Mitgeliefert werden außerdem:
-* fdfgen (https://github.com/ccnmtl/fdfgen) zum Editieren des Charakterbogens
-* cpdf (https://www.coherentpdf.com) zur Reduzierung der Dateigröße des Charakterbogens
-* hyphen (https://github.com/ytiurin/hyphen) für das korrekte Umbrechen von Textzeilen im Regelanhang
+#### Manuelle Installation
```

Comment:
Das könnte man auch in einem details Block verstecken: https://gist.github.com/pierrejoubert73/902cc94d79424356a8d20be2b382e1ab